### PR TITLE
update string matching for all languages

### PR DESCRIPTION
**Pull request by red031000**

### CODE REVIEW

File: pbrEngine/guiStateDistinguisher.py [hunk 1]
```diff
@@ -99,19 +99,49 @@ def distinguishStart(self, data):
 
 # start menu
 _map_start = {
-    "Wii Remote Control Sideways": PbrGuis.START_WIIMOTE_INFO,
+    "Wii Remote Control Sideways"                  : PbrGuis.START_WIIMOTE_INFO,
+    "Configuración horizontal del mando de Wii"    : PbrGuis.START_WIIMOTE_INFO,  # es
+    "Type de commandes horizontal"                 : PbrGuis.START_WIIMOTE_INFO,  # fr
+    "Steuerung C"                                  : PbrGuis.START_WIIMOTE_INFO,  # de
```
Comment (Felk) [inline pbrEngine/guiStateDistinguisher.py]: that sounds wrong, is "control c" supposed to be the title of the menu?

File: pbrEngine/guiStateDistinguisher.py [hunk 2]
```diff
@@ -99,19 +99,49 @@ def distinguishStart(self, data):
...
-    "Scegli una modalità di gioco"       : PbrGuis.START_MENU,  # it
-    "ゲームモードを　えらんでください"         : PbrGuis.START_MENU,  # ja
-
-    "Choose Options"                     : PbrGuis.START_OPTIONS,
-    "Save the changed options settings?" : PbrGuis.START_OPTIONS_SAVE,
-    "Announcer's Voice"                  : PbrGuis.START_VOICE,
-    "Colosseum Mode"                     : PbrGuis.START_MODE,
-    "Continue"                           : PbrGuis.START_SAVEFILE, # doesn't work, but relying on unstucker anyway
+    "Choose a Game Mode"                           : PbrGuis.START_MENU,
+    "Selección de modo de juego"                   : PbrGuis.START_MENU,  # es
```
Comment (Felk) [inline pbrEngine/guiStateDistinguisher.py]: can you please figure out a way of formatting that avoids diffs on otherwise unchanged lines?

**Review by Felk — CHANGES_REQUESTED** [review]

File: pbrEngine/guiStateDistinguisher.py (hunk 2 above)
Comment (red031000) [reply pbrEngine/guiStateDistinguisher.py]: the spacing is changed to line up better with longer lines, if that's not wanted I can revert it and only have the spacing change on the longer ones

Comment (Felk) [reply pbrEngine/guiStateDistinguisher.py]: ooooor, figure out a way of formatting that avoids diffs on otherwise unchanged lines. for example, a newline

**Review by Felk — APPROVED** [review]

**Pull request merged**